crates/libs/windows/src/core/heap.rs

```diff
@@ -32,20 +32,32 @@ pub unsafe fn heap_free(ptr: RawPtr) {
     }
 }
 
-/// Copy a slice of `T` into a freshly allocated buffer with an additional default `T` at the end.
+/// Copy len elements of an iterator of type `T` into a freshly allocated buffer with an additional default `T` at the end.
```

Comment:
No more default `T` at the end.

Reply:
Ah nice catch! Gonna fix it immediately.